CHANGELOG.md

```diff
@@ -16,6 +16,12 @@
 
 ---
 
+## v2.15.1
+
+- 🐛 [REPLAY-325] Add verbose error details on XHR transport failure ([#902](https://github.com/DataDog/browser-sdk/pull/902))
+- 👷 [RUMF-941] canary setup cdn ([#907](https://github.com/DataDog/browser-sdk/pull/907))
+- ♻️ extract domain context types to a separate file ([#901](https://github.com/DataDog/browser-sdk/pull/901))
```

Comment:
nit: don't hesitate to remove non customer facing tasks

Reply:
In this case, it would mean to remove all the list ? or maybe to replace it by something more vague?